frontend/src/concepts/modelRegistry/context/useModelRegistryNamespaceCR.ts

```diff
@@ -31,6 +31,10 @@ export const useModelRegistryNamespaceCR = (namespace: string, name: string): Fe
           // Not finding is okay, not an error
           return null;
         }
+        if (e.statusObject?.code === 403) {
+          // Forbidden is okay, not an error
+          return null;
+        }
```

Comment:
[I brought this up on a Slack thread here](https://redhat-internal.slack.com/archives/C06EJG57Z0X/p1722010483074919). The conclusion was that if we confirm I'm right that the `serverTimedOut` stuff is really not used anywhere in the ModelRegistryContext consumers, we should remove it and its related code there, and then since it will no longer call `useModelRegistryNamespaceCR` we can revert the change you have here @manaswinidas so that 403s are still thrown on the admin settings page.